torch_tvm/operators.cpp

```diff
@@ -35,6 +36,13 @@ const tvm::relay::TensorTypeNode getExprType(const tvm::relay::Expr& input) {
   return *checked_type;
 }
 
+TVMContext cpuContext() {
```

Comment:
looks like this is exactly the "static" cpu_ctx I was looking for above :) let's swap that instance with this

Reply:
sounds good